apps/docs/src/app/core/component-docs/list/list-docs.component.html

```diff
@@ -1,16 +1,78 @@
-<fd-docs-section-title [id]="'list-simple'" [componentName]="'list'">
-    Simple List
+<fd-docs-section-title id="list-simple" componentName="list">
+    Simple list
 </fd-docs-section-title>
-<description>A simple list. The list item can contain plain text or links.</description>
+<description>
+    To create a simple list extend your HTML list implementation with following directives: <code>fd-list</code>, <code>fd-list-item</code>,
+    <code>fd-list-title</code> directives.
+</description>
 <component-example>
     <fd-list-example></fd-list-example>
 </component-example>
 <code-example [exampleFiles]="simpleList"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'secondaryList'" [componentName]="'list'">
-    Secondary Item on List
+<fd-docs-section-title id="list-navigation" componentName="list">
+    Navigation list
+</fd-docs-section-title>
+<description>
+    For a standard list with navigation links add anchor elements with <code>fd-list-link</code> directive.
+    All items should be navigable.
+</description>
+<component-example>
+    <fd-list-navigation-example></fd-list-navigation-example>
+</component-example>
+<code-example [exampleFiles]="listNavigation"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="list-indicator" componentName="list">
+    Navigation indicator
+</fd-docs-section-title>
+<description>
+    If there is only a subset of list items that are navigable, they should be indicated with a navigation arrow.
+    Do not show arrows if all items are navigable, instead use a standard list with navigation.
+    To add navigation indicators, add <code>[navigationIndication]="true"</code> property to the
```

Comment:
This might be my bad, actually. Just missing an article here: "to add navigation indicators, add **a** <code>[navigationIndication]="true"</code> property to the..."